addons/web_editor/models/ir_ui_view.py

```diff
@@ -172,7 +172,12 @@ def _copy_field_terms_translations(self, record_from, name_field_from, record_to
             record_from[name_field_from],
             {lang: record_from.with_context(prefetch_langs=True, lang=lang)[name_field_from] for lang in langs if lang != lang_env}
         )
-        existing_translation_dictionary.update(extra_translation_dictionary)
+        for term, extra_translation_values in extra_translation_dictionary.items():
```

Comment:
Could you also mention this `overwrite=False` logic in the comment of this method? So that people won't abusively use it. And if someone really needs the feature `overwrite=False`, he can add it in the parameter himself.

Reply:
Hmm... Do you mean [something like this](https://github.com/odoo/odoo/pull/177960/commits/eb718ba448b240a937d48c4cfbe1fab19b61069b#diff-3fbc23861d612695a09633388940ac2c62aa2dadba54d4f55b29ba9fe72be70cR142-R152) :thinking: ?

Reply:
Yes, but I didn't expected it would be so complicated.
What about something like

Copy the model_terms translations from ``record_from.name_field_from`` to ``record_to.name_field_to`` for all activated languages if the term in ``record_to.name_field_to`` is untranslated (the term matches the one in the current language).

Reply:
LGTM :+1: